shell/browser/native_window_mac.mm

```diff
@@ -1075,6 +1075,12 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
       setSharingType:enable ? NSWindowSharingNone : NSWindowSharingReadOnly];
 }
 
+void NativeWindowMac::SetFocusable(bool focusable) {
+  [window_ setDisableKeyOrMainWindow:!focusable];
+  // Remove focus if the window had it.
+  Focus(false);
```

Comment:
This should only apply if `focusable` is false.

Reply:
I started to debug and it works if you put the setFocusable(false) in ready-to-show callback, show=false initially, and then call window.show(). 
Note that setFocusable(true) works in all cases. Just above limitation for the `false` case.

I am not sure this is fixable, also I noticed that NativeWindowMac::Focus(false) actually orders the window to the back instead of just removing the focus, so this existing function is already problematic. Any reason for that ? Thx!

Reply:
I just snooped around the [NSWindow documentation](https://developer.apple.com/documentation/appkit/nswindow/) and didn't find much regarding unfocusing windows. If this isn't possible via the AppKit APIs, should we simply document this OS-specific quirk?

cc @codebytere @zcbenz

Reply:
I don't think there is a way to unfocus window on macOS.